test/js/pageobjects/CrossDevice.js

```diff
@@ -0,0 +1,79 @@
+import Base from './BasePage.js'
+import {locale, verifyElementCopy} from '../utils/mochaw'
+
+class CrossDevice extends Base{
+
+  get crossDeviceTitle() { return this.$('.onfido-sdk-ui-PageTitle-titleSpan')}
+  get crossDeviceSubTitle() { return this.$('.onfido-sdk-ui-Theme-thickWrapper > div:nth-child(1)')}
```

Comment:
We shouldn't need to use the `nth-child` selector for the subtitle anymore. The dark days are over. :smile:

Reply:
This requires changes in Theme component.

Reply:
Looks like the class was not set because there is a reliance on the class existing in a component's corresponding `style.css` file.

Reply:
Done.